doc/api/cli.md

```diff
@@ -51,6 +51,37 @@ entry point, the `node` command will accept as input only files with `.js`,
 [`--experimental-wasm-modules`][] is enabled; and with no extension when
 [`--experimental-default-type=module`][] is passed.
 
+## Sub-commands
+
+### `node run <command> [<arguments>]`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development
+
+This runs an arbitrary command from a package.json's `"scripts"` object.
+If no `"command"` is provided, it will list the available scripts.
+
+By default, this command adds the current path appended by
+`node_modules/.bin` directory to the PATH in order to execute the binaries
+from dependencies.
```

Comment:
```suggestion
`node run` prepends `./node_modules/.bin`, relative to the current working directory,
to the `PATH` in order to execute the binaries from dependencies.
```

I assume it’s prepending rather than appending? We probably want local dependencies preferred over globally installed ones that share the same names.